STEP3/STEP3.swift

```diff
@@ -3,6 +3,53 @@
 //  CodeStarterCamp_Week1
 //
 //  Created by 한겨레 on 2022/06/23.
-//
 
-import Foundation
+import Foundation //기본적인 기능을 구현하기 위해서 필수적으로 import되어야하는 Framework정도?
+
+let bar = " | |" //bar모양은 어떤 빼빼로든 같으니까 상수
+let nudeDesign = "|0|" //누드디자인 상수
+
+//요구사항1. 길이에 따른 막대를 그리는 함수
+func makeBar(barLength: Int){
+    for count in 1...barLength{
+        print(bar)
+    }
+}
+//요구사항2. 몸통 ,토핑을 매개변수로 받아 토핑과 몸통을 그리는 함수.
+func makeBody(bodyShape: String,toppingShape: String)->String{
+    if(bodyShape == nudeDesign){
+        return " \(bodyShape)"
+    }//누드뺴뺴로
+    else if(toppingShape == ""){
+        return " \(bodyShape)"
+    }//토핑없는 빼빼로 앞 공백추가
+    else{
+        return "\(toppingShape)\(bodyShape)\(toppingShape)"
+    }//토핑이 있는 빼빼로는 앞뒤에 토핑을 추가
+}
+//요구사항3. 길이(Int)에 따라 몸통을 그리는 함수
+func drawBody(bodyHeight: Int,makeBody: String){
+    for count in 1...bodyHeight{
+        print(makeBody)
+    }
+}
```

Comment:
```suggestion
func drawBody(bodyHeight: Int, makeBody: String ){
    for count in 1...bodyHeight {
        print(makeBody)
    }
}
```
`Swift API Design GuideLines` ! ✨